index.html

```diff
@@ -309,11 +314,19 @@ <h3>
 
           <ol>
             <li>
+Remove the <code>did:web:</code> prefix.
+            </li>
+            <li>
 Replace ":" with "/" in the method specific identifier to obtain the fully
 qualified domain name and optional path.
             </li>
             <li>
-If the domain contains a port percent decode the colon.
+If the domain contains a port, percent decode the colon.
+            </li>
+            <li>
+Percent-decode each path fragment. (Note that although URI paths are technically
```

Comment:
I suggest removing this change suggestion (move to an issue / tackle in a subsequent pr).

Reply:
This is the main point of the PR tho..